### ISSUE TITLE
meteorMethods this object is window

### ISSUE THREAD

**Issue by lyquocnam** [jagi/meteor-astronomy#526]
in client i call:

```
const c = new Category({ name: 'xxx' });
    //console.log(c);
    c.create();
```

in Category.js (both folder);

```
...
meteorMethods: {

    create: function () {
      console.log(this);
      //this.set(payload);
      return this.save();
    },

  },
...
```

![image](https://cloud.githubusercontent.com/assets/8347826/19790962/daadb3c8-9c71-11e6-8b82-5ff48cdcb50e.png)

**Comment by jagi** [comment]
Are you sure that you're not messing with the `this` context? Show more code. I've checked this feature one more time and everything works, so it's probably bug on your side.

**Comment by lukejagodzinski** [comment]
Oh I forgot to mention that right now `meteorMethods` are only working for not new documents. But I will probably have to fix it.

**Comment by lyquocnam** [comment]
thanks @jagi good job 👍

**Comment by lukejagodzinski** [comment]
Fixed in version 2.2.4

**Comment by lyquocnam** [comment]
@thanks @jagi too fast men ! 👍
